dataplane/internal/engine/engine.go

```diff
@@ -78,10 +77,11 @@ type Engine struct {
 	// internalToExternalID is a map from the internal port id to it's corresponding external port.
 	internalToExternalID map[string]string
 	cancelFn             func()
+	createTables         bool
 }
 
 // New creates a new engine and sets up the forwarding tables.
-func New(ctx context.Context) (*Engine, error) {
+func New(ctx context.Context, createTables bool) (*Engine, error) {
```

Comment:
Add comment for why you might not want to create tables.

Reply:
added